src/metabase/query_processor/middleware/fetch_source_query_legacy.clj

```diff
@@ -0,0 +1,263 @@
+(ns ^:deprecated metabase.query-processor.middleware.fetch-source-query-legacy
```

Comment:
This is just the old `fetch-source-query` middleware copied over for the time being since some of its functions are used by other middleware that I haven't ported yet. Pls don't critique this because I didn't write any of this stuff (not recently, at least)